src/main/java/com/google/sps/servlets/utils/KeyRetriever.java

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets.utils;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class KeyRetriever {
+  private static String keyPath = System.getProperty("user.dir") + "/apiKey";
+
+  public static String getApiKey() {
```

Comment:
missing test for this function?

Reply:
Would this be something for a future pr?

Reply:
I think it's worth adding a unit test here to at least give us confidence that reading the API key on the backend works.

Reply:
+1 to tests, I think it'd be helpful

Reply:
Alright, I also think tests would be helpful. So, is the verdict adding Unit Tests to this PR rather than wait for a follow-up?